src/rules/javascript/index.js

```diff
@@ -1,15 +1,36 @@
-export * from './banned_identifiers';
-export * from './deprecated_entities';
-export * from './eval_string_arg';
-export * from './event_listener_fourth';
-export * from './global_require_arg';
-export * from './init_null_arg';
-export * from './low_level_module';
-export * from './mozindexeddb';
-export * from './mozindexeddb_property';
-export * from './only_prefs_in_defaults';
-export * from './opendialog_nonlit_uri';
-export * from './opendialog_remote_uri';
-export * from './shallow_wrapper';
-export * from './webextension_api';
-export * from './widget_module';
+import banned_identifiers from './banned-identifiers';
+import deprecated_entities from './deprecated-entities';
+import eval_string_arg from './eval-string-arg';
+import event_listener_fourth from './event-listener-fourth';
+import global_require_arg from './global-require-arg';
+import init_null_arg from './init-null-arg';
+import low_level_module from './low-level-module';
+import mozindexeddb from './mozindexeddb';
+import mozindexeddb_property from './mozindexeddb-property';
+import only_prefs_in_defaults from './only-prefs-in-defaults';
+import opendialog_nonlit_uri from './opendialog-nonlit-uri';
+import opendialog_remote_uri from './opendialog-remote-uri';
+import shallow_wrapper from './shallow-wrapper';
+import webextension_api from './webextension-api';
+import widget_module from './widget-module';
+
+
+export default {
+  rules: {
```

Comment:
The (probably naive) idea was that `rules/javascript` is more like a eslint-plugin (http://eslint.org/docs/developer-guide/working-with-plugins), eventually I'd like to move our json scanners and stuff to eslint processors but that's a step by step thing. Getting inspired by other eslint-plugins, e.g https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/src/index.js and I eventually wanted to get rid of our `const.ESLINT_RULE_MAPPING` and move that here to be more contained.